### PR TITLE
Issue #33: Prettier Hook for Formatting

### LINKED ISSUE #33: Prettier Hook for Formatting
For this issue, you will have to add a husky hook for pre-commit and pre-push. This will require you to install the package listed here: https://www.npmjs.com/package/husky

Here is a reference to a recent commit by Miguel doing the same: https://github.com/stevensblueprint/aad-admin/pull/152/files 

In our case, we only need npm run lint && npm run format

### PR DESCRIPTION
**Pull request by brrryry**
I finished an issue that required to use Husky in order to execute a hook before committing to the github repo.
Currently, Husky runs "npm run lint && npm run format" before committing.

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,8 +7,16 @@
...
 		"lint": "next lint",
+		"prepare": "husky",
 		"setup:users": "tinacms-next-auth setup",
-		"start": "next start"
+		"start": "next start",
+		"test": "echo No tests yet!"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "pnpm run lint && pnpm run format:check && pnpm run format",
```
Comment (miguel-merlin) [inline package.json]: Are we using pnpm? If so, do we need to install it

Comment (brrryry) [reply package.json]: Oh, good question! It was just copy-pasted from the example in the original issue, but we can modify/remove this as necessary. As stated in the issue, the only commands that should be run in the pre-commit file. I can remove this if you'd like.

Comment (miguel-merlin) [reply package.json]: Yeah you can change it to
```
npm run lint && npm run format:: check ...
```
pnpm is another package manager, but for this project we are just sticking with npm

Comment (ShawnAviles) [reply package.json]: TLDR 📖 : Yeah, just remove what you added for husky (lines 15-19) from the package.json. It's not needed.

To add on, here is a link to [pnpm docs](https://pnpm.io/motivation) if you want to read more about what it is in relation to npm. To clarify, we are _not_ using pnpm in this project. 

The basic structure of the example could've been used when creating the pre-commit hook. It just runs a regular npm script that must successfully run prior to making a commit. In the example, `pnpm run lint`, `pnpm run format:check` and `pnpm run format` are valid scripts inside the package.json file for that other repo. However in our repo, we do not have those scripts. We only have `lint` and `format` which are run using `npm run lint` and `npm run format` as you did in your pre-commit file. Technically, if we were using the older version of Husky you would need to change the values of 'pre-commit' and 'pre-push' to be `npm run lint && npm run format`.

So bottom line is that you're good to remove the husky stuff you added to the package.json (lines 15-19). [As of Husky v5 and above](https://typicode.github.io/husky/migrate-from-v4.html) (you're using v9), the `.husky/pre-commit` file is all you need 👍 

@brrryry Lmk if you any questions. Sorry for the lengthy response, just wanted to cover all the bases.
